internal/tools/changes/cmd/changetool/release.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/internal/tools/changes"
+)
+
+func releaseSubcmd(args []string) error {
+	if len(args) < 2 {
+		usage()
+	}
+
+	repo, err := changes.NewRepository(args[0])
+	if err != nil {
+		return fmt.Errorf("couldn't load repository: %v", err)
+	}
```

Comment:
I broke the positional CLI arguments into separate variables instead of using `args[x]` in multiple locations. I suppose these could also just be part of the `flags.FlagSet`s, but that seems like it would make the tool more difficult to use.